contest-service/service.bal

```diff
@@ -211,6 +230,29 @@ service /contestService on new http:Listener(9098) {
         }
     }
 
+    resource function get contests/[string contestId]/registrants()
+            returns registrantOut[]|MyInternalServerError|http:NotFound {
```

Comment:
Please remove the `MyInternalServerError` type from this package and use the `http:InternalServerError` instead